crates/nu-command/src/formats/from/toml.rs

```diff
@@ -56,6 +56,54 @@ b = [1, 2]' | from toml",
     }
 }
 
+fn convert_toml_datetime_to_value(dt: &Datetime, span: Span) -> Value {
+    match &dt.clone() {
+        toml::value::Datetime {
+            date: Some(_),
+            time: _,
+            offset: _,
+        } => (),
+        _ => return Value::string(dt.to_string(), span),
```

Comment:
If a parsed TOML datetime value type is `Local Time`, returns `Value::string`
https://docs.rs/toml/latest/toml/value/struct.Datetime.html

Reply:
I'm not sure if returning a string where a date is expected is right either, but we can try it to see if others object.